src/prebid.js

```diff
@@ -503,13 +503,17 @@ pbjsInstance.requestBids = (function() {
     events.emit(REQUEST_BIDS);
     const cbTimeout = timeout || config.getConfig('bidderTimeout');
     logInfo('Invoking $$PREBID_GLOBAL$$.requestBids', arguments);
+    if (adUnitCodes != null && !Array.isArray(adUnitCodes)) {
+      adUnitCodes = [adUnitCodes];
```

Comment:
Was this another bug?

Reply:
yes, but it's weirder than it looks. we are filtering ad units by `adUnitCodes.includes(adUnit.code)`, and strings can include each other, so passing `adUnitCodes: 'some-code'` would include `some-code` as expected, but also `some` and `code`.

Reply:
Oh 🤯that's a good catch